client/my-sites/checkout/upsell-nudge/difm-upsell/index.jsx

```diff
@@ -0,0 +1,187 @@
+			</header>
+		);
+	}
+
+	body() {
+		const { translate } = this.props;
+		return (
+			<>
+				<h2 className="difm-upsell__header">
+					{ translate(
+						'Skip to the launch line: {{br/}} Let our WordPress experts build your website',
+						{
+							components: { br: <br /> },
+						}
+					) }
+				</h2>
+
+				<div className="difm-upsell__column-pane">
+					<div className="difm-upsell__column-content">
+						<p>
+							{ translate(
+								'Get up and running quickly by letting our WordPress experts build and design your website from scratch.'
+							) }
+						</p>
+						<p>
+							{ translate(
+								'Whether you’re launching a store, a business, or even online courses, we’ll help you stand out online with a beautiful, functional, secure website that’s built just for you.'
+							) }
+						</p>
+						<p>
+							{ translate(
+								'You’ll work with a dedicated engagement manager throughout the entire project, ensuring that your vision is carried through from start to finish. All for a one-time fee.'
+							) }
+						</p>
+						<p>{ translate( 'Our premium website building service is perfect for:' ) }</p>
+						<ul className="difm-upsell__checklist">
+							<li className="difm-upsell__checklist-item">
+								<Gridicon icon="checkmark" className="difm-upsell__checklist-item-icon" />
+								<span className="difm-upsell__checklist-item-text">
+									{ translate( 'Online stores', {
+										comment:
+											"This is a type of business listed on the 'Website building service' offer page",
+									} ) }
+								</span>
+							</li>
+							<li className="difm-upsell__checklist-item">
+								<Gridicon icon="checkmark" className="difm-upsell__checklist-item-icon" />
+								<span className="difm-upsell__checklist-item-text">
+									{ translate( 'Educational websites', {
+										comment:
+											"This is a type of business listed on the 'Website building service' offer page",
+									} ) }
+								</span>
+							</li>
+							<li className="difm-upsell__checklist-item">
+								<Gridicon icon="checkmark" className="difm-upsell__checklist-item-icon" />
+								<span className="difm-upsell__checklist-item-text">
+									{ translate( 'Professional services', {
+										comment:
+											"This is a type of business listed on the 'Website building service' offer page",
+									} ) }
+								</span>
+							</li>
+							<li className="difm-upsell__checklist-item">
+								<Gridicon icon="checkmark" className="difm-upsell__checklist-item-icon" />
+								<span className="difm-upsell__checklist-item-text">
+									{ translate( 'Advanced tailored solutions', {
+										comment:
+											"This is a type of business listed on the 'Website building service' offer page",
+									} ) }
+								</span>
+							</li>
+						</ul>
+
+						<p>
+							{ translate(
+								'Simply tell us a little bit more about your website, and we’ll get back to you in 2-3 business days.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Fill out the application form and tell us a little bit more about your website. We’ll get back to you in 2-3 business days.' )` *ES Score: 6*